src/sales/order-status-notification.md

```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
 1. Set **Customer Order Status Notification** to `Enable`.
 
 1. When complete, click **Save Config**.
+
+## Configure New Order Email Notification
+
+1. On the Admin sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand **Sales** and choose **Sales Email** underneath.
+
+1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Order** section and enable the order email notification.
+
+1. Set **New Order Confirmation Email Sender** to one of the following:
+
+    - General Contact
+    - Sales Representative
+    - Customer Support
+
+1. Choose the **New Order Confirmation Template** for `Customers` and `Guests`.
```

Comment:
List the three options in full -
New Order (Default)
New Pickup Order 
New Pickup Order For Guest